bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/AccessibleToolItemBuilder.java

```diff
@@ -0,0 +1,61 @@
+package org.eclipse.ui.texteditor;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.widgets.ToolItem;
+
+/**
+ * Builder for ToolItems for {@link AccessibleToolBars}.
+ *
+ * @since 3.17
```

Comment:
Since 3.17 here but Since 3.18 in `AccessibleToolbar`? Odd, please check.

Reply:
Both should be 17 which is the current version!

Reply:
I have set both to be in version 3.17 in https://github.com/eclipse-platform/eclipse.platform.ui/commit/0c0d22bc8f0ea13eb6e1124c89747f7a4f02e266